com.unity.toonshader/Documentation~/MatCap.md

```diff
@@ -50,70 +45,80 @@ MatCap color blending mode. Multiply or Add.
 ## Scale MatCap UV
 Scaling UV of MatCap Map.
 
-<img src="images/ScaleMatCapUV.gif" height="256">
+<video title="A close-up on the blonde hair of a chibi-style character model. The brighter yellow specular highlight changes from a thin line to a wider highlight." src="images/ScaleMatCapUV.mp4" width="auto" height="auto" autoplay="true" loop="true" controls></video>
 
 
 ## Rotate MatCap UV
 Rotating UV of MatCap Map.
 
-<img src="images/RotateMatCapUV.gif" height="256">
+<video title="A close-up on the blonde hair of a chibi-style character model. The yellow specular highlight rotates from left to right and back." src="images/RotateMatCapUV.mp4" width="auto" height="auto" autoplay="true" loop="true" controls></video>
+
 
 ## Stabilize Camera Rolling
 Stabilize Camera rolling when capturing materials with camera.
 
-| Stabilize Camera Rolling Off  | Stabilize Camera Rolling On |
-| -- | --|
-| <img src="images/StabilizerOff.gif" height="256">| <img src="images/StabilizerOn.gif" height="256"> |
+<video title="A close-up on the face of a chibi-style character model. When the camera rotates, the specular highlights stay in the same place in the camera view." src="images/StabilizerOff.mp4" width="auto" height="auto" autoplay="true" loop="true" controls></video>
+Stabilize Camera Rolling disabled.
+
+<video title="The same close-up. When the camera rotates, the specular highlights stay in the same place on the hair." src="images/StabilizerOn.mp4" width="auto" height="auto" autoplay="true" loop="true" controls></video>
+Stabilize Camera Rolling enabled.
 
 
 ## Normal Map Specular Mask for MatCap
 If enabled, gives a normal map specifically for MatCap. If you are using MatCap as speculum lighting, you can use this to mask it.
 
-| Normal Map Specular Mask Off  | Normal Map Specular Mask On |
-| -- | --|
-| <img src="images/MatCapNormalMapOff.png" height="256">| <img src="images/MatCapNormalMapOn.png" height="256"> |
+![A view from above a chibi-style character model with blonde hair and brighter yellow specular highlights.](images/MatCapNormalMapOff.png)<br/>
+Normal Map Specular Mask disabled.
+
+![The same view. The individual hairs now have highlights.](images/MatCapNormalMapOn.png)<br/>
+Normal Map Specular Mask enabled.
 
 
 ### Normal Map
 A texture that dictates the bumpiness of the material.
 
-| Normal Map Texture Example | 
-| -- |
-| <img src="images/HairNormalMask.png" height="256">|
+
+![A grayscale texture of fine hair.](images/HairNormalMask.png)<br/>
+An example of a Normal Map texture.
+
 
 ### Rotate Normal Map UV
```

Comment:
In [the result](https://yamato-artifactviewer.prd.cds.internal.unity3d.com/0ea85a0e-3f8b-490b-8317-0bd27b086420%2Fdocumentation%2FPackage_Doc/manual/index.html) with the content corresponding to [e62c5e9](https://github.com/Unity-Technologies/com.unity.toonshader/pull/424/commits/e62c5e927a4418b25b8a4f01b723dceb9bdc5301).
Looks ### is rendered in the page.

Reply:
Again I think this is an issue with the preview. It looks ok in my local preview:

![image](https://github.com/user-attachments/assets/6d30cc2d-7d93-40e5-bfed-8a20309c087d)